src/ServerFactory.php

```diff
@@ -149,6 +150,18 @@ public function getTempDir()
         }
     }
 
+    /**
+     * Get cache path callable.
+     *
+     * @return \Closure|null Cache path callable.
+     */
+    public function getCachePathCallable()
+    {
+        if (isset($this->config['cache_path_callable'])) {
```

Comment:
A single line will suffice `return $this->config['cache_path_callable'] ?? null;`